task9/repository/meteorsClient.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+import { config } from "../config/config.js";
+
+export const getMeteorData = async(startDate, endDate) => {
+    try {
+        const nasaUrl = `${config.nasaMeteorUrl}&start_date=${startDate}&end_date=${endDate}`;
+        const response = await axios.get(nasaUrl);
```

Comment:
it is possible to use some alternative syntax
```
axios.get(config.nasaMeteorUrl, {
    params: {
      start_date: startDate,
      end_date: endDate
    }
  })
```

Reply:
done